tests/storage/export.go

```diff
@@ -66,6 +73,32 @@ const (
 	pvcNotFoundReason = "pvcNotFound"
 
 	proxyUrlBase = "https://virt-exportproxy.%s.svc/api/export.kubevirt.io/v1alpha1/namespaces/%s/virtualmachineexports/%s%s"
+
+	tlsKey   = "tls.key"
+	tlsCert  = "tls.crt"
+	testKey  = "test"
+	testCert = `-----BEGIN CERTIFICATE-----
+MIIDJTCCAg2gAwIBAgIUYhSmCxHywX8qmhCPSZweno9MP8cwDQYJKoZIhvcNAQEL
+BQAwIjEgMB4GA1UEAwwXdm1leHBvcnQtcHJveHkudGVzdC5uZXQwHhcNMjIwNjEz
+MTcwNTA4WhcNMjMwNjEzMTcwNTA4WjAiMSAwHgYDVQQDDBd2bWV4cG9ydC1wcm94
+eS50ZXN0Lm5ldDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBALrkCWbD
+z2i/QAOQXOX6bk0Q3wq6wRz+qHtr9mu93FQrZUcMzYpr/dLMaW18PVes4RjMuqjt
+cbTjgGIwoS3VFCyub9wB1cro+0avf8cnl7Z10RPA7i6b2UV800HcwlmrbGr29kN2
+E2x2PajAAjoAqAaqu1rhP1ua7xynpWiDk64pXVDxdPUUvyoDD6eL3KqSqA1X/iys
+fG6s+ypL540mIDLR5rxtvvdD4J7hktT+UIinbEE7Q9DwoGJ6xKqYagOb2CLymNLZ
+ruJ3vmiTqd/2HXGI+4xe2fR6v23CRhLIeRBIzznIJOwBGe/YutGCIHa35AATlhwS
+HzFagYwOATIMvt8CAwEAAaNTMFEwHQYDVR0OBBYEFJVH/4uLVQ8gPqa6x7D46Jga
+jikIMB8GA1UdIwQYMBaAFJVH/4uLVQ8gPqa6x7D46JgajikIMA8GA1UdEwEB/wQF
+MAMBAf8wDQYJKoZIhvcNAQELBQADggEBAKPg9lwsRMpJ6wR1hR/1GWFVqJsb7t7G
+nr0SPsYBYzOkIfMtIK7kRfLQgMRTNOpvyC13zkDeAFjejr+Ovl7d4ONLGP/7qcj/
+dLN0Tas2m0pMJjxJa40WYU+XlnO47R/HtYnrn7RMIZ2TUroisf2oiEUJ36989mp5
+ArFLLJzozIrpAP5lHThpJjQCs6wM/d7daasRBfXucGPMvXcUnVmGo+2pUjTourpG
+NH18UMKg4B0B1Gs6LsQVYrQerP9zI14p3nu3ieBFEUT2wuERycbHLwCH+mC4HpbD
+pS/x4m6CgALv0lrhHpffeOc8lncNEzkZbRpCG1NBbPA6mFLtklcuER8=
+-----END CERTIFICATE-----`
```

Comment:
If this can be generated in the test, I think that would be ideal.  Otherwise, we have to worry about expiry and names of things changing

Reply:
You make a good point, I didn't want to go through the effort of generating the cert. but I should just to ensure I have no expiry issues.

Reply:
So thought about this a bit more, and there is actually no place we check the dates on the certificate. The code just takes whatever is in the secret or config map, finds the one that has a matching domain name/alt name, and puts that in the cert field. No expire check at all. Maybe we should check, not sure. I am still going to write the code to generate the cert in case in the future we will care about the expiry.